### PR TITLE
#407 - add selected_partners for direct selection project

### PR DESCRIPTION
**Pull request by lkolacz**

### CODE REVIEW

File: backend/unpp_api/apps/project/serializers.py [hunk 1]
```diff
@@ -327,6 +344,13 @@ def get_application(self, obj):
             return ApplicationPartnerSerializer(qs.get()).data
         return None
 
+    def get_direct_selected_partners(self, obj):
```
Comment (elyk) [inline backend/unpp_api/apps/project/serializers.py]: Is this for a different story?

Comment (lkolacz) [reply backend/unpp_api/apps/project/serializers.py]: I'm not follow You in this question. What do You mean by different story?

**Comment by lkolacz** [comment]
@elyk 
I can add if statement for status (for agency) - make it as Serializer Method Field .. and return empty or msg (status is visible only for agency member) for partner member

File: backend/unpp_api/apps/project/serializers.py (hunk 1 above)
Comment (elyk) [reply backend/unpp_api/apps/project/serializers.py]: Nevermind. Did two stories in one PR. All good.

**Comment by elyk** [comment]
@lkolacz ok. It is for @marcindo to comment on if easier to keep as just `status` field for `Application` and depending on who the user is (Partner or Agency), we would display different type of statuses (new ones added here for Partner and existing for Agency)

**Comment by lkolacz** [comment]
@marcindo @elyk 
selected partners are moved to right serializer (agency only)
abot #505 I believe that we shouldn't merge this two tickets

**Pull request merged**